src/OpenTelemetry/Metrics/Configuration/MeterFactory.cs

```diff
@@ -1,4 +1,4 @@
-// <copyright file="MeterFactory.cs" company="OpenTelemetry Authors">
+﻿// <copyright file="MeterFactory.cs" company="OpenTelemetry Authors">
```

Comment:
what changed in this line?

Reply:
Looks like UTF8 encoding issue, will try re-save with other encoding.